src/nunit.testmodel/nunit.testmodel.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Nunit.Gui</RootNamespace>
-    <AssemblyName>nunit.gui.core</AssemblyName>
+    <AssemblyName>nunit.testmodel</AssemblyName>
```

Comment:
So i've noticed lately that `nunit.testmodel` and `nunit.uikit` are different naming conventions (personally i prefer it this way) compared to `nunit-gui` and `mock-assembly` It throws me off sorta,...and i'm not sure why. Maybe it's the way windows sorts the folders.

ugh.. kids need me. brb for more reviewing.

Reply:
Do you mean the hyphens? I use dot-separated components to indicate that the rhs is subordinate to the lhs. So, uikit and testmodel are parts of NUnit. I use hyphens when somebody or thing has a hyphenated name - circular logic, I know. 😄 Like Jean-Jacques or nunit-console. Of course nunit-console and nunit-gui can be construed either way - as a single name or as in "the console component of nunit". But for mock-assembly, there's no doubt that it's a single name, that assembly is not a component of mock, that is. Anyway, it makes sense to me. 😄

Reply:
Also, I've never liked dot-separated names for an exe.